src/librbd/cache/pwl/Types.h

```diff
@@ -281,13 +278,16 @@ struct WriteLogCacheEntry {
 
 struct WriteLogPoolRoot {
   #ifdef WITH_RBD_RWL
+  uint32_t crc = 0;
+  uint64_t write_num = 0;        /* decide which root space to write.
+                                  * If the number is even, write to 0~2k;
+                                  * otherwise, write 2~4k */
```

Comment:
add more comments for this policy: to make superblock update atomic you use two superblock.